buildpackage_forMendix.ps1

```diff
@@ -0,0 +1,15 @@
+Param(
```

Comment:
Could you please rename it to `mx_buildpackage.ps1`?

Reply:
done